src/shared/context/TableContext/tableReducer.ts

```diff
@@ -104,5 +96,15 @@ const actionMap = {
 } as const
 
 export const tableReducer = (state: TableState, action: TableAction) => {
-  return actionMap[action.type]?.(state, action) ?? state
+  const debugEnabled = getEnvFlag('debug_logs')
+  if (debugEnabled) {
+    console.log('action', action)
+    console.log('state', state)
+  }
+  const newState = actionMap[action.type]?.(state, action) ?? state
+  if (debugEnabled) {
+    console.log('newState', newState)
+  }
+
+  return newState
```

Comment:
if this is used just here, I'd get rid of it for now. if needed, in the future, I'd use something like [loglevel](https://www.npmjs.com/package/loglevel) and set the log level in the feature flag. for now let's remove it

Reply:
these logs in particular are super useful when dealing with the table, but a performance hindrance in production.
so yea, we can discuss the best course of action